.github/workflows/publish-ghcr-container.yaml

```diff
@@ -0,0 +1,73 @@
+name: Publish multiarch images on ghcr.io
+on:
+  push:
+    tags:
+      - '*'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  publish:
+    name: Build and push Spilo multiarch images
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      packages: 'write'
+    defaults:
+      run:
+        shell: bash
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+
+    - name: Install flake8 and docker-compose
+      run: python -m pip install flake8 docker-compose==1.17.1
+
+    - name: Derive spilo image name
+      id: image
+      working-directory: postgres-appliance
+      run: |
+        PGVERSION=$(sed -n 's/^ARG PGVERSION=\([1-9][0-9]*\).*$/\1/p' Dockerfile)
+        IMAGE="${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}-$PGVERSION:${GITHUB_REF/refs\/tags\//}"
+        echo "::set-output name=NAME::$IMAGE"
```

Comment:
Can we switch to [the new format](https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter) here? The `set-output` command is deprecated.

Reply:
Nice :), didn't know about it yet. Looks interesting!